Top-Level Requirements.md

```diff
@@ -77,10 +77,12 @@
 * 8.8 Upon completing the mining procedure, the mining subsystem **shall**:
     * 8.8.1 Define an “obstacle” representing the digging hole and send it to the navigation subsystem.
     * 8.8.2 Return the mining arm to the starting position.
+* 8.9 The mining subsystem **shall** provide an ability to move the digging arm to a predefined dumping position to enable bin movement without interference
```

Comment:
this is a little messy, can we strike 8.9 and reword 8.8.2 to cover it?

The goal we are trying to accomplish is to clean up after we finish, and that's the job of requirement 8.8.

Would this work:
     * 8.8.2 Return the mining arm to an ending position clear of the range of motion of the dumping bin.

Reply:
Yeah, sounds good.